### ISSUE TITLE
[Windows] server giving "Service Unavailable" error 

### ISSUE THREAD

**Issue by russ1974x** [owncloud/client#4616]
### Actual behaviour
[Note - I posted this on the forums first, moderator told me to post here as a bug]

On one Windows 10 machine, my account logs in to ownCloud, sees all of the folders, can see all of the changes from the server, but when it tries to sync, gives this error.  "Error downloading xxxxxxxxx - server replied:Service Unavailable. Operation cancelled."

The error lists a specific file, but if I delete that file, the error then appears for the next file.  Works fine on several other Windows 10 PCs (all using the same user, and hence same data), and works fine from the web client.  Have tried a full reinstall of ownCloud client a few times.  Can't think of anything else to try other than a full Windows 10 OS format/reinstall, as it's working fine on my other machines!

### Server configuration
Ubuntu (latest), ownCloud server 8.2.3, latest PHP, but this is only affecting one client so don't think it's a server side issue!

### Client configuration
Client version: Windows 2.1.1 (latest)

Operating system:
Windows 10 Pro

OS language:
English

Installation path of client:
Default (C:\Program Files (x86)\ownCloud)

### Logs
I have cut & pasted the log entries during the error below (I have removed the username and server for privacy) - anything here that might give a clue? Next step is a complete format and reinstall of Windows 10 so hoping to avoid that!!

03-29 08:20:59:335 0x4b377e0 OCC::Folder::estimateState: bool OCC::Folder::estimateState(QString, csync_ftw_type_e, OCC::SyncFileStatus*) Folder has error "Card Spending"
03-29 08:20:59:336 0x4b377e0 OCC::SocketApi::fileStatus: Folder estimated status for "Card Spending" to "ERROR"
03-29 08:20:59:336 0x4b377e0 OCC::SocketApi::sendMessage: SocketApi: Sending message: "STATUS:ERROR:C:\Users\[XX--HIDDEN--XX]\ownCloud\Card Spending"
03-29 08:20:59:337 0x4b377e0 OCC::SyncEngine::slotItemCompleted: void OCC::SyncEngine::slotItemCompleted(const OCC::SyncFileItem&, const OCC::PropagatorJob&) "" INSTRUCTION_NONE 0 ""
03-29 08:20:59:337 0x4b377e0 OCC::Folder::estimateState: bool OCC::Folder::estimateState(QString, csync_ftw_type_e, OCC::SyncFileStatus*) Folder has error ""
03-29 08:20:59:338 0x4b377e0 OCC::SocketApi::fileStatus: Folder estimated status for "" to "ERROR"
03-29 08:20:59:338 0x4b377e0 OCC::SocketApi::sendMessage: SocketApi: Sending message: "STATUS:ERROR:C:\Users\[XX--HIDDEN--XX]\ownCloud\"
03-29 08:20:59:339 0x4b377e0 OCC::SyncEngine::slotItemCompleted: void OCC::SyncEngine::slotItemCompleted(const OCC::SyncFileItem&, const OCC::PropagatorJob&) "" INSTRUCTION_NONE 0 ""
03-29 08:20:59:339 0x4b377e0 OCC::Folder::estimateState: bool OCC::Folder::estimateState(QString, csync_ftw_type_e, OCC::SyncFileStatus*) Folder has error ""
03-29 08:20:59:340 0x4b377e0 OCC::SocketApi::fileStatus: Folder estimated status for "" to "ERROR"
03-29 08:20:59:340 0x4b377e0 OCC::SocketApi::sendMessage: SocketApi: Sending message: "STATUS:ERROR:C:\Users\[XX--HIDDEN--XX]\ownCloud\"
03-29 08:20:59:341 0x4b377e0 OCC::SyncJournalDb::walCheckpoint: void OCC::SyncJournalDb::walCheckpoint() took 0 msec
03-29 08:20:59:341 0x4b377e0 OCC::SyncJournalDb::commitInternal: void OCC::SyncJournalDb::commitInternal(const QString&, bool) Transaction commit "All Finished." 
03-29 08:20:59:341 0x4b377e0 OCC::SyncEngine::finalize: CSync run took 15417
03-29 08:20:59:342 0x4b377e0 OCC::BandwidthManager::~BandwidthManager: virtual OCC::BandwidthManager::~BandwidthManager()
03-29 08:20:59:347 0x4b377e0 OCC::Folder::slotSyncFinished: - client version 2.1.1 (build 5837) Qt 5.4.0 SSL OpenSSL 1.0.2f 28 Jan 2016
03-29 08:20:59:347 0x4b377e0 OCC::Folder::slotSyncFinished: -> SyncEngine finished with ERROR, warn count is 2
03-29 08:20:59:371 0x4b377e0 OCC::Folder::bubbleUpSyncResult: Processing result list and logging took 22 Milliseconds.
03-29 08:20:59:372 0x4b377e0 OCC::Folder::bubbleUpSyncResult: OO folder slotSyncFinished: result: 3
03-29 08:20:59:374 0x4b377e0 OCC::Folder::slotSyncFinished: ** error Strings: ("Error downloading https://owncloud2.[XX--HIDDEN--XX]/remote.php/webdav/Card Spending/Archive/April 2015/002.pdf - server replied: Service Unavailable", "Operation canceled", "Card Spending/Archive/April 2015/002.pdf: Error downloading https://owncloud2.[XX--HIDDEN--XX]/remote.php/webdav/Card Spending/Archive/April 2015/002.pdf - server replied: Service Unavailable", "Card Spending/Archive/April 2015/003.pdf: Operation canceled")
03-29 08:20:59:374 0x4b377e0 OCC::Folder::slotSyncFinished: * owncloud csync thread finished with error
03-29 08:20:59:375 0x4b377e0 OCC::Folder::slotSyncFinished: the last 61 syncs failed
03-29 08:20:59:375 0x4b377e0 OCC::Folder::estimateState: bool OCC::Folder::estimateState(QString, csync_ftw_type_e, OCC::SyncFileStatus*) Folder has error ""
03-29 08:20:59:376 0x4b377e0 OCC::SocketApi::fileStatus: Folder estimated status for "" to "ERROR"
03-29 08:20:59:376 0x4b377e0 OCC::SocketApi::sendMessage: SocketApi: Sending message: "STATUS:ERROR:C:\Users\[XX--HIDDEN--XX]\ownCloud\"
03-29 08:20:59:377 0x4b377e0 OCC::SocketApi::sendMessage: SocketApi: Sending message: "UPDATE_VIEW:C:\Users\[XX--HIDDEN--XX]\ownCloud\"
03-29 08:20:59:379 0x4b377e0 OCC::ownCloudGui::slotSyncStateChange: Sync state changed for folder "https://owncloud2.[XX--HIDDEN--XX]/remote.php/webdav/" : "Error"
03-29 08:20:59:380 0x4b377e0 OCC::FolderWatcher::pathIsIgnored: * Ignoring file "C:/Users/[XX--HIDDEN--XX]/ownCloud/.owncloudsync.log"
03-29 08:20:59:580 0x4b377e0 OCC::FolderMan::slotFolderSyncFinished: <===================================== sync finished for "https://owncloud2.[XX--HIDDEN--XX]/remote.php/webdav/"
03-29 08:21:17:757 0x4b377e0 OCC::ConnectionValidator::checkAuthentication: # Check whether authenticated propfind works.
03-29 08:21:17:757 0x4b377e0 OCC::AbstractNetworkJob::start: !!! OCC::PropfindJob created for "https://owncloud2.[XX--HIDDEN--XX]" + "/" "OCC::ConnectionValidator"
03-29 08:21:20:739 0x4b377e0 OCC::AbstractNetworkJob::start: !!! OCC::PropfindJob created for "https://owncloud2.[XX--HIDDEN--XX]" + "/" "OCC::QuotaInfo"
03-29 08:21:41:882 0x4b377e0 OCC::Folder::slotRunEtagJob: * Trying to check "https://owncloud2.[XX--HIDDEN--XX]/remote.php/webdav/" for changes via ETag check. (time since last sync: 42 s)
03-29 08:21:41:885 0x4b377e0 OCC::FolderMan::slotRunOneEtagJob: Scheduling "https://owncloud2.[XX--HIDDEN--XX]/remote.php/webdav/" to check remote ETag
03-29 08:21:41:885 0x4b377e0 OCC::AbstractNetworkJob::start: !!! OCC::RequestEtagJob created for "https://owncloud2.[XX--HIDDEN--XX]" + "/" "OCC::Folder"
03-29 08:21:50:783 0x4b377e0 OCC::AbstractNetworkJob::start: !!! OCC::PropfindJob created for "https://owncloud2.[XX--HIDDEN--XX]" + "/" "OCC::QuotaInfo"

**Comment by russ1974x** [comment]
Ok so I decided to go ahead and format the machine, full clean reinstall of Windows 10 but I still get the same error!  Now it's really confusing as I am running the client on two other machines without a problem.  Must be something server side I guess??

**Comment by guruz** [comment]
```Service Unavailable``` is generated by the server, so it is not client related.
Have you checked the websrver's error log? Have you checked data/owncloud.log (when setting the logging to verbose in the web UI)
Is this uploading or downloading? (the message in our code is wrong, it always says "downloading")

**Comment by russ1974x** [comment]
You were right - there something weird with the files.   I took a local copy of the data on one of my working clients, and then deleted all of the data from all clients and the web client, made sure they all synced as "empty".   Then I copied the files back in on one client, they all synced fine - including the machine that was playing up.   So not a client issue, something odd had clearly happened to the data on the server.  All working fine now - certainly something I did!
